src/structures/interfaces/InteractionResponses.js

```diff
@@ -85,6 +88,8 @@ class InteractionResponses {
       files,
     });
     this.replied = true;
+
+    return options.fetchReply ? this.fetchReply() : undefined;
```

Comment:
```suggestion
    if (options.fetchReply) return this.fetchReply();
```
looks a bit better but probably doesnt make a difference either way

Reply:
This causes eslint's `consistent-return` rule to fail

Reply:
oh :angry: